build.gradle.kts

```diff
@@ -105,7 +106,7 @@ subprojects {
 }
 
 apply(from = rootProject.file(".buildscript/binary-validation.gradle"))
-
+apply(plugin = "com.github.ben-manes.versions")
```

Comment:
Why was this added?

Reply:
This allows us to see which libraries need updating with the following command:

```
gradle dependencyUpdates -Drevision=release
```

Reply:
We're already using one plugin to manage dependencies, I'm very reluctant to add a second.